src/common/test/testUtils.tsx

```diff
@@ -9,3 +10,11 @@ export const mountWithProvider = (children: ReactElement) =>
 
 export const shallowWithProvider = (children: ReactElement) =>
   shallow(<Provider store={store}>{children}</Provider>);
+
+/* eslint-disable  @typescript-eslint/no-explicit-any */
+export const updateWrapper = async (wrapper: any) => {
```

Comment:
> updateWrapper

AFAIK this is used to test components that fetch data with apollo. Could use a library like [`wait-for-expect`](https://github.com/TheBrainFamily/wait-for-expect) instead, as is suggested in the official [Apollo docs](https://www.apollographql.com/docs/react/development-testing/testing/)?

Reply:
That actually was first thing that I tried, but I ended up going with this solution instead. Everything I tried with `wait-for-expect` ended up in an error `When testing, code that causes React state updates should be wrapped into act(...):`. Maybe I give this package a new go when I have more time to play around with it.